ena_portal_api/ena_handler.py

```diff
@@ -166,7 +165,7 @@ def get_default_connection_headers():
     return {
         "headers": {
             "Content-Type": "application/x-www-form-urlencoded",
-            "Accept": "*/*",
+            "accept": "*/*",
```

Comment:
This one should be "Accept" -> https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept
Is the ENA API complaning?